scripts/split-e2e-tests-v2.ts

```diff
@@ -72,6 +72,8 @@ const WINDOWS_SMOKE_TESTS = [
   'src/__tests__/env-1.test.ts',
   // export and pull
   'src/__tests__/export-pull-a.test.ts',
+  // feature flags
+  'src/__tests__/feature-flags.test.ts',
```

Comment:
this is one of 8 tests that's sensitive to that failure, so adding to windows smoke test list.